### PR TITLE
fix(aptos): amount balance

### PR DESCRIPTION
**Pull request by joaoccmartins**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Remove the delegated assets from the aptos dashboard. Notice this is being merged to `feat/integrate-aptos`.

### ❓ Context

- **JIRA or GitHub link**: [LIVE-14963]

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14963]: https://ledgerhq.atlassian.net/browse/LIVE-14963?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx:1]: I think we may even be able to remove this component, and use the generic one

File: apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx [hunk 1]
```diff
@@ -81,19 +80,6 @@ const AccountBalanceSummaryFooter = ({ account }: Props) => {
           <Discreet>{spendableBalance}</Discreet>
         </AmountValue>
       </BalanceDetail>
-      <BalanceDetail>
-        <ToolTip content={<Trans i18nKey="aptos.delegation.delegatedInfoTooltip" />}>
```
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx:85]: We may need to remove the translation linked to this key :)

**Review by hedi-edelbloute — CHANGES_REQUESTED** [review]

File: apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx (hunk 1 above)
Comment (joaoccmartins) [reply apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx:85]: I've removed the translation unit.

File: apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx
Comment (joaoccmartins) [reply apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx:1]: Removed.

**Review by hedi-edelbloute — APPROVED** [review]

**Pull request merged**